packages/app/pages/course/[cid]/invite.tsx

```diff
@@ -21,10 +21,6 @@ export default function Invite(): ReactElement {
   const cardMetaDescription = `This course is managed by ${course?.organizationCourse.name}`
 
   useEffect(() => {
-    if (!profile) {
-      localStorage.setItem('lastVisited', window.location.href)
```

Comment:
We had bunch of issues with this code, so I moved it to be backend and store it as cookies